services/movie_session.py

```diff
@@ -40,5 +41,21 @@ def update_movie_session(
     movie_session.save()
 
 
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    taken_seats = []
+
+    tickets = Ticket.objects.filter(movie_session__id=movie_session_id)
+
+    for ticket in tickets:
+        taken_seats.append(
+            {
+                "row": ticket.row,
+                "seat": ticket.seat
+            }
+        )
+
+    return taken_seats
```

Comment:
Use method `value` of QuerySet here

Reply:
Thank you very much I didn't think of that before